packaging/README.md

```diff
@@ -1 +1,22 @@
 # PLUMgrid Libnetwork Plugin Packaging
+
+Follow the steps below to build a libnetwork plugin debian package:
+
+- Create a go workspace and clone the libnetwork plugin repo (follow main README steps).
+
+- Navigate to the packaging directory and run the deb creation script:
+
+```
+$ cd $GOPATH/src/github.com/libnetwork-plugin/packaging
+$ chmod +x build_libnetwork_deb.sh
+$ ./build_libnetwork_deb.sh
+```
+This will make the source files and create a new debian package under *$GOPATH/src/github.com/libnetwork-plugin/*
+
+- The debian can now be installed with:
+
+```
+$ dpkg -i libnetwork_<version>_all.deb
+```
+
+**Note:** this package has a dependency on docker-engine and golang being pre-installed.
```

Comment:
That line only checks for those depedencies already being installed. Since both of these packages, or their required versions, are not available on default upstream mirrors, we can't have this package install them automatically. The option is to either make sure they are preinstalled or take out the dependencies from this entirely. Let me know what you want to go with.